build.zig

```diff
@@ -45,6 +45,47 @@ pub fn build(b: *std.build.Builder) void {
         lint_step.dependOn(&run_cmd.step);
     }
 
+    {
+        const benchmark = b.addExecutable("eytzinger_benchmark", "src/eytzinger_benchmark.zig");
+        benchmark.setTarget(target);
+        benchmark.setBuildMode(.ReleaseSafe);
+        const run_cmd = benchmark.run();
+
+        const step = b.step("eytzinger_benchmark", "Benchmark array search");
+        step.dependOn(&run_cmd.step);
+    }
+
+    {
+        const benchmark = b.addExecutable("benchmark_ewah", "src/ewah_benchmark.zig");
+        benchmark.setTarget(target);
+        benchmark.setBuildMode(.ReleaseSafe);
+        const run_cmd = benchmark.run();
+
+        const step = b.step("benchmark_ewah", "Benchmark EWAH codec");
+        step.dependOn(&run_cmd.step);
+    }
+
+    {
+        const benchmark = b.addExecutable(
+            "benchmark_segmented_array",
+            "src/lsm/segmented_array_benchmark.zig",
+        );
+        benchmark.setTarget(target);
+        benchmark.setBuildMode(.ReleaseSafe);
+        benchmark.setMainPkgPath("src/");
+        const run_cmd = benchmark.run();
+
+        const step = b.step("benchmark_segmented_array", "Benchmark SegmentedArray search");
+        step.dependOn(&run_cmd.step);
+    }
+
+    // Executable which generates src/c/tb_client.h
+    const tb_client_header_generate = blk: {
```

Comment:
`zig build tb_client_header` would be a thing only if we have a `b.addStep()` for it. ATM, it's only an internal step that's a dependency for `zig build test` and `zig build tb_client`. Doesn't have to be for the latter, but updating the header in source when modifying tb_cilent serves as a reminder to update it for the clients too (maybe automate this as well since all clients are in tree?)

Reply:
Ah, good point. Yes, we should automate updating the clients too, but you can open a github issue for that to defer it to a future PR.